editor/src/core/workers/parser-printer/parser-printer-bugs.spec.ts

```diff
@@ -285,7 +285,23 @@ export default function () {
 }
 `
 
-    testParseThenPrintWithoutUids(spreadCode, spreadCode)
+    testParseThenPrintWithoutUids('/index.js', spreadCode, spreadCode)
+  })
+
+  it('#1773 - Handles imports which relate to the same file but have differing paths.', () => {
+    const spreadCode = `import * as React from 'react'
+import { FirstComponent } from './components.js'
```

Comment:
do I read it correctly that we are reserving the user's multiple imports? that's great!

Reply:
Attempts to, but doesn't guarantee it is probably the better way to describe it.